examples/cookbook/plugins/picture_using_camera/lib/main_step3.dart

```diff
@@ -55,6 +56,7 @@ class TakePictureScreenState extends State<TakePictureScreen> {
     );
 
     // Next, initialize the controller. This returns a Future.
+    _initializeControllerFuture = _controller.initialize();
```

Comment:
I'm curious what the intent of this future is? Will it be used by a `FutureBuilder` to update the UI once the camera controller is initialized?